Formula/threesdk.rb

```diff
@@ -0,0 +1,31 @@
+class Threesdk < Formula
+  desc "Software development kit for JumpscaleX"
+  homepage "https://wiki.threefold.io/"
```

Comment:
This homepage doesn't look related to "Threesdk"

Reply:
this was the wiki for it but replaced with the direct one